.github/workflows/android-apk.yml

```
@@ -0,0 +1,62 @@
+name: Android APK Build
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)